### DISCUSSION TITLE
Anchor points to control short jump stitches

### DISCUSSION THREAD

**Discussion by MadTooler** [inkstitch/inkstitch#2371] [Q&A]
Since I really don't know what I am doing with this needle and thread stuff, I have been looking around for a technique to see if it is a good or terrible practice and what it may be called in the world of embroidery.    While running a machine that does not cut jump stitches, and working with small fonts with very short jumps between lettering, I was thinking maybe I could make some anchor points to jump to between letters that would allow for longer and easier cutting of jumps and better control where the jumps are routed to avoid stitching over them.  The anchors could be a single or maybe two stitches to allow for easy removal.  My first attempt within inkstitch is with a 6mm straight line path with params for a running stitch with manual stitch mode.  That gives a single stitch at the start and end with the 6mm length being long enough to easily remove like a short basting stitch.

Is this a bad idea?

I have to assume this has been done before.  If so, what is it called and is there a feature in inkscape that does it better than my attempt?

Thanks

**Comment by MadTooler** [comment]
This is an example of what I was describing.  While this 2023 is not too terribly small, 7.5mm tall, and the jumps don't really have routing issues, extending them up to the anchors could make it easier to cut them.  At least for me.  What do you think?

![image](https://github.com/inkstitch/inkstitch/assets/16729670/76d683bb-df74-41a9-8885-2677ee7363de)

**Reply by lexelby** [reply]
Oh yes, this is an excellent idea! I used to do this when I had a machine that couldn't cut jump stitches.  I would snip the bobbin thread on the back and then pull the top thread taut before snipping so that I got it trimmed perfectly.  Works great.

I found that I could create a single-stitch path by editing a path using the XML Editor tool in Inkscape.  I'm not sure whether it was a valid SVG path, but I basically created a path that was a single point.  It was invisible, so I set a starting marker to make it visible.  Ink/Stitch treated it as a single stitch.  I think I had to set it as manual stitch.

Somewhere in our Issues section, there's a feature request I created describing all this and suggesting that it would be cool if Ink/Stitch could place these trim helpers automatically, taking into account the order things stitch in so that the jumps don't get sewn over later on.

**Comment by MadTooler** [comment]
Thanks for the reply.

> I found that I could create a single-stitch path by editing a path using the XML Editor tool in Inkscape. I'm not sure whether it was a valid SVG path, but I basically created a path that was a single point. It was invisible, so I set a starting marker to make it visible. Ink/Stitch treated it as a single stitch. I think I had to set it as manual stitch.

My original attempt was a single point that inkscape could see, but that actually creates a small circle with multiple points and possible fill.  With your method, it keeps the extra stitches minimal, but I do not like the extra "invisible" work that I may forget about when I come back to revisit the design (maybe that's just me).  

I just test ran the 2023 I noted initially on a Brother SE1900.  I was afraid it would auto perform lock stitches on the manual jumps, but it did not.  It ran the way I had hoped, and it was very easy to remove and trim.  Of course, I since I was only testing and not too concerned about exactly how the jumps at the numbers stitched, it did satin stitch over my poorly positioned jumps a little.  

Sounds like there isn't a specific feature for this technique.  I was kind of thinking of a visual command option, but what I did with manual stitching was super easy.  The only chore was properly moving them in the layer tree.

Are you aware of the proper designation/name of this technique?
